tests/discrete_gauss.rs

```diff
@@ -0,0 +1,68 @@
+use num_bigint::{BigInt, BigUint};
```

Comment:
New files should have copyright notices.

Reply:
You mean the `SPDX-License-Identifier: MPL-2.0`, right?

Reply:
Yup :)